os/binfmt/binfmt_loadbinary.c

```diff
@@ -105,28 +126,34 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 		bin = (FAR struct binary_s *)kmm_zalloc(sizeof(struct binary_s));
 		if (!bin) {
 			berr("ERROR: Failed to allocate binary_s\n");
-			errcode = ENOMEM;
+			errcode = -ENOMEM;
```

Comment:
There seems to be some inconsistency with the assignment of the error value to `errcode`
It is assigned a positive value at some places and then passed on to `set_errno` to be taken care by.
At other points, it is assigned a negative Error Signal value, eg. line 113, 168.